src/app/permissions/common-permission-response.ts

```diff
@@ -0,0 +1,7 @@
+import { UserPermissionScope } from '../users/user-permission-scope';
+import { User } from '../users/user';
+
+export interface CommonPermissionResponse {
+  scope: UserPermissionScope;
```

Comment:
I agree with the `scope` field as the part of `CommonPermissionResponse` at the moment. Though we need to keep in mind the scopes of `GlobalUserPermissionResponse` and `UserPermission` can be different. For instance the possible values of the `scope` field of `GlobalUserPermissionResponse` are `A`, `B`, `C`. And the possible values of the `scope` field of `UserPermission` are `A`, `C`, `D`.

From my point of view it would make sense to consider the two different enumerations for personal and global permissions e.g. `GlobalUserPermissionScope`, `UserPermissionScope` like it was done on the server side:
https://github.com/T-Systems-MMS/skoop-server/blob/master/src/main/java/com/tsmms/skoop/user/UserPermissionScope.java
https://github.com/T-Systems-MMS/skoop-server/blob/master/src/main/java/com/tsmms/skoop/user/GlobalUserPermissionScope.java

Reply:
Thank you. Done